src/main/java/com/banking/app/bankingApp/Users/CreateUser.java

```diff
@@ -0,0 +1,90 @@
+package com.banking.app.bankingApp.Users;
```

Comment:
move this to location request/user, do it also for other user related request models